index.js

```diff
@@ -61,14 +73,27 @@ class RC433EtekcitySwitch {
       });
   }
 
-  toggleSwitch(signal) {
+  toggleNext(switchObject) {
+    // get next todo item
+    var todoItem = todoList.shift();
+    var signal = todoItem['signal'];
+    var callback = todoItem['callback'];
+    // send signal
```

Comment:
Der Konsistenz halber fände ich in Zeile 78,79 und 80 `let` besser, was meinst du?